Documentation/compatibility/ServiceBase-doesnt-propagate-OnStart-exceptions.md

```diff
@@ -0,0 +1,26 @@
+## ServiceBase doesn't propagate OnStart exceptions
+
+### Scope
+
+Minor
+
+### Version Introduced
+
+4.7.1
+
+### Source Analyzer Status
+
+NotPlanned
+
+### Change Description
+
+Exceptions from services that fail to start will now be propagated from ServiceBase.Run
```

Comment:
What is the previous (4.7 and earlier) behavior? No exception thrown? Also, is this an opt-in feature (you opt-out by default)? In that case:

In the .NET Framework 4.7 and earlier versions, exceptions thrown on service startup are not propagated to the caller of <xref::System.ServiceProcess.Servicebase.Run%2A>. 

Starting with applications that target the .NET Framework 4.7.1, the runtime propagates exceptions to  <xref::System.ServiceProcess.Servicebase.Run%2A> for services that fail to start.